### PR TITLE
update 1.1p to 1.2i in matching_fof

### PR DESCRIPTION
**Pull request by yymao**
This PR updates the `matching_fof` tutorial:
- object catalog from 1.1p to 1.2i
- truth catalog from 1.1 to 1.2
- extragalactic catalog (protoDC2) from 2.1.2 to 3.0
- "coadd" --> "object"

This PR partly addresses #19.

### CODE REVIEW

**Comment by yymao** [comment]
I've updated the one of last plots of this matching tutorial where we compare the magnitudes in the object catalog and the extragalactic catalog. The update is to use the cModel magnitude for the object catalog. This resolves the strange offset in the original case. 

Here's the new plot:

image

**Comment by yymao** [comment]
@rmandelb @reneehlozek: I think @danielsf and @drphilmarshall probably have very limited bandwidth right now so if you can help find someone else to review this PR that would be quite helpful. Thanks!

**Comment by drphilmarshall** [comment]
Thanks @yymao - your diagnosis is correct :-) @jiwoncpark Would you mind helping me out by reviewing Yao's updated notebook, please? Thanks!

File: tutorials/matching_fof.ipynb [hunk 1]
```diff
@@ -366,7 +357,7 @@
    "source": [
     "# load redshift and shear parameters from the extragalactic catalog, only for galaxise that are already in `matched_gals`\n",
     "extragalactic_data = extragalactic_cat.get_quantities(\n",
-    "    ['galaxy_id', 'mag_i', 'ellipticity_true', 'shear_1', 'shear_2'],\n",
+    "    ['galaxy_id', 'mag_i_lsst', 'ellipticity_true', 'shear_1', 'shear_2'],\n",
     "    filters=[(lambda x: np.in1d(x, matched_gals['object_id'].values, True), 'galaxy_id')]\n",
```
Comment (reneehlozek) [inline tutorials/matching_fof.ipynb]: Most of the changes seem simple, but I suppose one check would be that this redefinition of mag_i_lsst is backwards propogated?

**Comment by yymao** [comment]
In extragalactic catalogs, `mag_i` is simply an alias for `mag_i_lsst`, as setting the LSST filters being the default seems to make sense.

On the other hand, in the object catalogs, `mag_i` is the PSF magnitude (as opposed to the cModel magnitude `mag_i_cModel`). There's no `mag_i_lsst` as everything is in LSST filters.

Because of this difference, I thought using `mag_i` in extragalactic catalogs in this case may be confusing, so I change it to `mag_i_lsst`. But either name actually gives you the same quantity (for extragalactic catalogs).

**Pull request merged**